<ui version="4.0" >
 <class>AddAccountFormClass</class>
 <widget class="QWidget" name="AddAccountFormClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AddAccountForm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Server:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editServer" >
     <property name="text" >
      <string>irc.freenode.net</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editPort" >
     <property name="text" >
      <string>6667</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Nick:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editNick" />
   </item>
   <item>
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Real Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editRealName" />
   </item>
   <item>
    <widget class="QLabel" name="labelPasswd" >
     <property name="text" >
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editPasswd" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkboxSavePasswd" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="text" >
      <string>Save password</string>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>334</width>
       <height>68</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
